### PR TITLE
refactor: rename package name

### PR DESCRIPTION
**Pull request by septs**
| Old package name                                | New package name                   |
| --------------------------------------- | -------------------------- |
| @dimensiondev/maskbook                  | @masknet/extension         |
| @dimensiondev/cli                       | @masknet/cli              |
| @dimensiondev/contracts                 | @masknet/contracts        |
| @dimensiondev/dashboard                 | @masknet/dashboard        |
| @dimensiondev/icons                     | @masknet/icons            |
| @dimensiondev/maskbook-theme            | @masknet/theme            |
| @dimensiondev/mask-plugin-infra         | @masknet/plugin-infra     |
| @dimensiondev/plugin-example            | @masknet/plugin-example   |
| @dimensiondev/external-plugin-previewer | @masknet/external-plugin-previewer |
| @dimensiondev/maskbook-shared           | @masknet/shared           |
| @dimensiondev/web3-shared               | @masknet/web3-shared      |
| @dimensiondev/maskbook-storybook-shared | @masknet/storybook-shared |

The `@masknet` npm organization, controlled by @yisiliu

### CODE REVIEW

**Comment by Jack-Works** [comment]
I suggest rename to `@mask-network/mask`

**Comment by yisiliu** [comment]
I've reserved Maskbook, masknet, masknetwork. Let me try to appeal to get the mask from the npm support and hold this pr for a while.

**Comment by septs** [comment]
> I suggest rename to `@mask-network/mask`

I hope to shorten this package name

**Comment by Jack-Works** [comment]
Please rename `@masknet/maskbook` to `@masknet/mask` or `@masknet/extension`

**Comment by septs** [comment]
> Please rename `@masknet/maskbook` to `@masknet/mask` or `@masknet/extension`

done

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,18 +28,15 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: "**/dist/"
-          key: ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
-          restore-keys: |
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}
+          key: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
+          restore-keys: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}
```
Comment (Jack-Works) [inline .github/workflows/build.yml:32]: This change removes the ability to a fallback cache when the lock file has changed, which is tolerable in the tsc case. I'd prefer to revert this change.

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -51,12 +51,10 @@ jobs:
...
           # actions/cache will not upload changes in cache if primary key hits
           # by adding date to the primary key, we can ensure the cache updates on the first build of the day
-          key: ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
+          key: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
           # Allow fallback to cache of different dependencies but not allowing fallback to different tsconfig
           # because that might indicates a structural/flags changes in tsc emit.
-          restore-keys: |
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}
+          restore-keys: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}
```
Comment (Jack-Works) [inline .github/workflows/linters.yml:57]: This change removes the ability to a fallback cache when the lock file has changed, which is tolerable in the tsc case. I'd prefer to revert this change.

Comment (septs) [reply .github/workflows/linters.yml:57]: https://github.com/DimensionDev/Maskbook/runs/2891126512?check_suite_focus=true

```
hashFiles('**/tsconfig.json')
```

triggle timeout......

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]
